src/node_crypto.cc

```diff
@@ -4800,7 +4800,7 @@ void SignOneShot(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(signature.ToBuffer().ToLocalChecked());
 }
 
-Verify::Verify(Environment* env, v8::Local<v8::Object> wrap) :
+Verify::Verify(Environment* env, Local<Object> wrap) :
```

Comment:
Feel free to move the `:` to the following line like we do elsewhere for consistency, since you’re already here :)